charts/tigera-operator/templates/tigera-operator/02-tigera-operator.yaml

```diff
@@ -98,12 +98,20 @@ spec:
             - operator
           args:
             - -bootstrap-crds
+          envFrom:
+            - configMapRef:
+                name: kubernetes-services-endpoint
+                optional: true
         # Install any v3 API resources provided in the calico-resources ConfigMap.
         - name: create-initial-resources
           image: {{.Values.calicoctl.image}}:{{.Values.calicoctl.tag}}
           env:
             - name: DATASTORE_TYPE
               value: kubernetes
+          envFrom:
+            - configMapRef:
+                name: kubernetes-services-endpoint
+                optional: true
```

Comment:
not sure if we need this here for the calicoctl container (does it use the services endpoint configmap?), but it probably doesn't hurt to have it...